ast/compile.go

```diff
@@ -345,6 +350,7 @@ func NewCompiler() *Compiler {
 		{"CheckDeprecatedBuiltins", "compile_state_check_deprecated_builtins", c.checkDeprecatedBuiltins},
 		{"BuildRuleIndices", "compile_stage_rebuild_indices", c.buildRuleIndices},
 		{"BuildComprehensionIndices", "compile_stage_rebuild_comprehension_indices", c.buildComprehensionIndices},
+		{"BuildRequiredCapabilities", "compile_stage_build_required_capabilities", c.buildRequiredCapabilities},
```

Comment:
Can you add this one to https://github.com/open-policy-agent/opa/commit/544fd0329d3918f6137b953e6a6007d6fd0af952#diff-067a842cc5e1a348551f4c7656120105acedafb2faac6e7b7ff57b75c5349570R1494? I don't think we'll care about the min req caps for wasm (and IR)

Reply:
I can add it though I wonder if knowing the built-ins required by the policy may be useful in some cases?

Reply:
Yeah it might be time to expose the desired stages in some way. 🤔